Doc/tutorial/controlflow.rst

```diff
@@ -246,6 +249,172 @@ at a more abstract level.  The :keyword:`!pass` is silently ignored::
    ...     pass   # Remember to implement this!
    ...
 
+
+.. _tut-match:
```

Comment:
Note that we now have two slightly diverging versions of the same introduction to match statements in the docs (three if you count Appendix A of PEP 636) -- the other is in what's new in 3.10. We don't have to fix this now but I think it would be good if the texts didn't arbitrarily differ from each other. Either the same text should be included in both places (possibly with a different intro and outro), or what's new should be much shorter and just point here. Again, doesn't need to be addressed today, but something to keep in mind. /CC @willingc

Reply:
It makes good sense to bring the two closer together. I would err on the side of clarity over length in the What's New as it's what most folks will read before or after the PEP tutorial.